domain-server/src/DomainServer.cpp

```diff
@@ -3060,6 +3071,26 @@ void DomainServer::initializeExporter()
     }
 }
 
+void DomainServer::initializeMetadataExporter() {
+    static const QString ENABLE_EXPORTER = "metaverse.enable_metadata_exporter";
+    static const QString EXPORTER_PORT = "metaverse.metadata_exporter_port";
+
+    bool isMetadataExporterEnabled = _settingsManager.valueOrDefaultValueForKeyPath(ENABLE_EXPORTER).toBool();
+    int metadataExporterPort = _settingsManager.valueOrDefaultValueForKeyPath(EXPORTER_PORT).toInt();
+
+    if (metadataExporterPort < MIN_PORT || metadataExporterPort > MAX_PORT) {
+        qCWarning(domain_server) << "Metadata exporter port " << metadataExporterPort << " is out of range.";
```

Comment:
```suggestion
        qCWarning(domain_server) << "Metadata exporter port" << metadataExporterPort << "is out of range.";
```
Spaces are automatically included between items.